hotshot-query-service/src/availability/data_source.rs

```diff
@@ -233,6 +234,8 @@ where
         hash: TransactionHash<Types>,
     ) -> Fetch<TransactionQueryData<Types>>;
 
+    async fn get_state_cert(&self, epoch: u64) -> Fetch<StateCertQueryData<Types>>;
```

Comment:
This doesn't seem to be called anywhere? To hook this up we would need to add a new GET endpoint handler in `hotshot-query-service/src/availability.rs` to handle the `get_state_cert` request and call this function

Reply:
I forgot to add the handler, done in [9c85bbb](https://github.com/EspressoSystems/espresso-network/pull/2917/commits/9c85bbb25d6e0a9af2266e39b2f5644933337adb)